polyfactory/factories/sqlalchemy_factory.py

```diff
@@ -124,8 +125,13 @@ def should_column_be_set(cls, column: Any) -> bool:
     @classmethod
     def get_type_from_column(cls, column: Column) -> type:
```

Comment:
We should document these. I don't think any are exposed in the reference documentation because of lack of docstrings